### ISSUE TITLE
[java] ConstructorCallsOverridableMethod false positive when calling super method

### ISSUE THREAD

**Issue by Andy-2639** [pmd/pmd#1718]
Affects PMD Version: 6.12.0
Rule: [ConstructorCallsOverridableMethod](https://pmd.github.io/latest/pmd_rules_java_errorprone.html#constructorcallsoverridablemethod)
Running PMD through: PMD Eclipse Plugin version 4.1.0

Code Sample demonstrating the issue:

```java
public class A {
    public void doA() {}
}

public class B extends A {
    public B() {
        super.doA();  // <-- PMD: Overridable method 'doA' called during object construction
    }

    @Override
    public void doA() {
        throw new UnsupportedOperationException();
    }
}
```

Description:

In B's constructor, PMD reports that an overridable method is called. However, it is a super method call. So I think it is a false positive.
